xbmc/pvr/PVRManager.cpp

```diff
@@ -509,6 +509,9 @@ void CPVRManager::Process()
   // Reinit playbackstate
   m_playbackState->ReInit();
 
+  // Update startup GUI process with playback and group state
```

Comment:
Please remove this comment. Events are introduced to achieve loose coupling of components, so as a publisher you do not have to know what will happen once the event is published. Or will you adjust this comment whenever a new subscriber for this event is implemented? ;-)